src/main/drivers/dma.h

```diff
@@ -67,12 +72,25 @@ typedef enum {
     DMA_MAX_DESCRIPTORS
 } dmaIdentifier_e;
 
-#define DMA_MOD_VALUE       8
-#define DMA_MOD_OFFSET      0
+#define DMA_DEVICE(x)       (((x-1) / 8) + 1)
+#define DMA_DEVICE_INDEX(x) (((x-1) % 8))
 #define DMA_OUTPUT_INDEX    0
 #define DMA_OUTPUT_STRING   "DMA%d Stream %d:"
+#define DMA_INPUT_STRING    "DMA%d_ST%d"
+
+#define DEFINE_DMA_CHANNEL(d, s, f, r) { \
+    .dma = d, \
+    .ref = d ## _Stream ## s, \
+    .stream = s, \
+    .irqHandlerCallback = NULL, \
+    .flagsShift = f, \
+    .irqN = d ## _Stream ## s ## _IRQn, \
+    .rcc = r, \
```

Comment:
RCC can be inferred from DMA (either in this macro, or better in init code)

Reply:
Have inferred - there is at max two possibilities for all targets.